clueless/static/js/controllers.js

```diff
@@ -200,9 +207,23 @@ class GameHub {
         this.gameState.players[i].character.position = message.location;
       }
     }
+    // I don't think a toast is needed for this since you can see the move happen, but we could always add one.
   }
   handleMsgSuggestion(message) {
-    this.gamePanel.showToast('Suggestion: ' + JSON.stringify(message));
+    // Get which player has won by character id
+    var player;
+    for (var i = 0; i < gameHub.gameState.players.length; i++) {
+      if (gameHub.gameState.players[i].character.id == message.player) {
+        player = gameHub.gameState.players[i];
+      }
+    }
+    var room = WitnessItem_fromType(message.room, WitnessType.ROOM);
+    var suspect = WitnessItem_fromType(message.suspect, WitnessType.CHARACTER);
+    var weapon  = WitnessItem_fromType(message.weapon, WitnessType.WEAPON);
+
+    var msg = player.display_name + ' made a suggestion: ' + room.name + ', ' + suspect.name + ', ' + weapon.name;
```

Comment:
I think this should be 
` ... + suspect.name + ' in ' + room.name + ' with the ' + weapon.name;`
Reads well, and is the way its done done in the real game.

Reply:
Excellent suggestion! Thanks! Adding